db/migrate/20210927021406_add_production_to_actor.rb

```diff
@@ -0,0 +1,6 @@
+class AddProductionToActor < ActiveRecord::Migration[6.1]
+  def change
+    add_column :actors, :production_id, :integer
```

Comment:
It would make more sense for productions to be a join table that links the actor to many movies and shows.